rpc/query_server.go

```diff
@@ -1054,6 +1056,25 @@ func (s *QueryServer) EthGetBalance(address eth.Data, block eth.BlockHeight) (et
 	return eth.EncBigInt(*amount.Int), nil
 }
 
+// https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_getStorageAt
+func (s *QueryServer) EthGetStorageAt(local eth.Data, position string, block eth.BlockHeight) (eth.Data, error) {
```

Comment:
We shouldn't ignore the block height parameter, should accept "latest" or a number matching `snapshot.Block().Height`, return `fmt.Errors("unable to get storage at height %v", block)` for any another value.

Reply:
An empty `block` parameter is also fine, in that case it should be assumed to be whatever the snapshot height is (i.e. "latest").